templates/zerver/api/changelog.md

```diff
@@ -10,6 +10,11 @@ below features are supported.
 
 ## Changes in Zulip 4.0
 
+**Feature level 39**
+
+* [`GET /users/<user_id_or_email>/presence`]: Added support for user_id
+in `get_user_presence` endpoint.
```

Comment:
Let's write this as "Added support for passing a `user_id` to identify the target user." -- no need to repeat the endpoint name via the less-official `get_user_presence` naming schema (which doesn't appear elsewhere in user-facing documentation)

Also, this needs a rebase an update to a new feature level, since we've merged other API migrations since you first wrote this.

Finally, a few of the details in your commit message, like "This is ready for review" and "Added changes to changelog" are unnecessary and would be better removed.

Reply:
Ok